docs/source/nextflow-workflow.rst

```diff
@@ -135,6 +135,163 @@ Example ``sample_sheet.csv``
     +-------------------------------------------------+------------------------------------------------------+---------------------------------------------------------------------+
 
 
+Quick start
+###########
+The following is a condensed summary of steps required to get Autometa installed, configured and running. There are links throughout to the appropriate documentation sections that can provide more detail if required.
+
+Installation
+************
+For full installation instructions, please see the :ref:`Installation` section
```

Comment:
🐞 😕  The cross-reference is self-referencing because the same header is used in this document as is used in `installation.rst`

Since this section is titled `Installation`, the cross-reference is referencing itself. To get around this, we can add a custom anchor, e.g.

### Example different document to be referenced:

```rst
.. _anchor-to-place-before-where-you-want-to-reference:

===================
Section being referenced
===================
```

### Example usage in current document doing the cross-referencing

```rst
:ref:`anchor-to-place-before-where-you-want-to-reference`
```

### Suggested change for `Autometa/docs/source/installation.rst`

```rst
.. _installation-page:

============
Installation
============
```

### Example usage in current document doing the cross-referencing

```suggestion
For full installation instructions, please see the :ref:`installation-page` section
```